diff --git a/node_modules/@codetrix-studio/capacitor-google-auth/android/build.gradle b/node_modules/@codetrix-studio/capacitor-google-auth/android/build.gradle
index 4bb66ea..101d4aa 100644
--- a/node_modules/@codetrix-studio/capacitor-google-auth/android/build.gradle
+++ b/node_modules/@codetrix-studio/capacitor-google-auth/android/build.gradle
@@ -7,8 +7,8 @@ ext {
 
 buildscript {
     repositories {
-        jcenter()
         google()
+        mavenCentral()
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:8.2.1'
@@ -45,11 +45,9 @@ android {
 
 repositories {
     google()
-    jcenter()
     mavenCentral()
 }
 
-
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation project(':capacitor-android')
